fuse_constraints/CMakeLists.txt

```diff
@@ -35,7 +35,47 @@ catkin_package(
 ###########
 ## Build ##
 ###########
-add_compile_options(-Wall -Werror)
+
+# Disable warnings about array bounds with -Wno-array-bounds until gcc 12 fixes this bug:
```

Comment:
Would it make sense to file a ticket with this error and then reference it here?

Reply:
Good point. I can look into that